# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/telepathy-glib/hotfix-vala.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- telepathy-glib-0.99.11/telepathy-glib/TelepathyGLib-1.metadata.vanilla	2026-09-28 01:41:27.630297763 +0300
+++ telepathy-glib-0.99.11/telepathy-glib/TelepathyGLib-1.metadata	2026-09-28 01:41:32.562188060 +0300
@@ -1 +1,2 @@
 TelepathyGLib name="TelepathyGLib"
+BaseConnection.get_object_path name="get_tp_object_path"
